Consuming recipes from ConanCenter (#220)
conan-io/conan-io.github.io: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown

---
layout: post
comments: false
title: "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?"
meta_title: "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?" 

> meta_description: "Learn how to safely consumer recipes using your very own ArtifactoryCE"

Author: ```suggestion
meta_description: "Learn how to safely consume public recipes using your very own ArtifactoryCE"
```

---

<script type="application/ld+json">
{ "@context": "https://schema.org", 
 "@type": "TechArticle",
 "headline": "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?d",

>  "alternativeHeadline": "Learn how to setup your enviroment to consumer recipes from ConanCenter",

Author: ```suggestion
 "alternativeHeadline": "Learn how to setup your environment to consume recipes from ConanCenter",
```

 "image": "https://docs.conan.io/en/latest/_images/frogarian.png",
 "author": "Christopher McArthur, Conan Developer Advocate", 
 "genre": "C/C++", 
 "keywords": "c c++ package manager conan release", 
 "publisher": {
    "@type": "Organization",
    "name": "Conan.io",
    "logo": {
      "@type": "ImageObject",
      "url": "https://media.jfrog.com/wp-content/uploads/2017/07/20134853/conan-logo-text.svg"
    }
},

>  "datePublished": "2022-08-21",

Author: ```suggestion
 "datePublished": "2022-09-01",
```

>  "description": "Improved download tool to support getting files from the local file system, support for components in MSBuildDeps, new MesonDeps generator, improved CMakePresets integration and lots of fixes to ease Conan 2.0 migration.",

Reviewer: This part is still here from another blogpost

Author: ```suggestion
 "datePublished": "2022-08-21",
 "description": "Learn how to safely consume recipes from ConanCenter using your very own ArtifactoryCE.",
```

>  "description": "Improved download tool to support getting files from the local file system, support for components in MSBuildDeps, new MesonDeps generator, improved CMakePresets integration and lots of fixes to ease Conan 2.0 migration.",

Author: ```suggestion
 "description": "Learn how to safely consume recipes from ConanCenter using your very own ArtifactoryCE.",
```
